beamer/tests/health/test_check.py

```diff
@@ -0,0 +1,307 @@
+import dataclasses
+import pickle
+import time
+
+from brownie import ZERO_ADDRESS
+from hexbytes import HexBytes
+from web3.types import Wei
+
+from beamer.agent.events import ClaimMade, RequestCreated, RequestFilled
+from beamer.agent.typing import ChainId, FillId, Nonce, Termination, TokenAmount
+from beamer.health.check import (
+    ChainEventMap,
+    Context,
+    NotificationTypes,
+    Transfer,
+    analyze_transfer,
+    create_transfers_object,
+)
+from beamer.tests.agent.unit.utils import (
+    ADDRESS1,
+    BLOCK_NUMBER,
+    CLAIM_ID,
+    CLAIMER_STAKE,
+    REQUEST_ID,
+)
+from beamer.tests.agent.utils import make_address
+
+TARGET_CHAIN_ID = ChainId(42161)
+SOURCE_CHAIN_ID = ChainId(10)
+TARGET_TOKEN_ADDRESS = make_address()
+SOURCE_TOKEN_ADDRESS = make_address()
+AGENT_ADDRESS = make_address()
+EVIL_AGENT_ADDRESS = make_address()
+
+request = RequestCreated(
+    request_id=REQUEST_ID,
+    target_chain_id=TARGET_CHAIN_ID,
+    source_token_address=SOURCE_TOKEN_ADDRESS,
+    target_token_address=TARGET_TOKEN_ADDRESS,
+    source_address=ADDRESS1,
+    target_address=ADDRESS1,
+    amount=TokenAmount(123),
+    nonce=Nonce(123),
+    valid_until=Termination(123),
+    block_number=BLOCK_NUMBER,
+    tx_hash=HexBytes(b"1"),
+    chain_id=SOURCE_CHAIN_ID,
+)
+
+fill = RequestFilled(
+    request_id=REQUEST_ID,
+    chain_id=TARGET_CHAIN_ID,
+    fill_id=FillId(b"1"),
+    source_chain_id=SOURCE_CHAIN_ID,
+    target_token_address=TARGET_TOKEN_ADDRESS,
+    filler=AGENT_ADDRESS,
+    amount=TokenAmount(100),
+    block_number=BLOCK_NUMBER,
+    tx_hash=HexBytes(b"1"),
+)
+
+claim = ClaimMade(
```

Comment:
`request`, `fill` and `claim` should be all-uppercase to be consistent with the rest of module-level vars.